app/routes/forum/components/ThreadDetail.tsx

```diff
@@ -0,0 +1,80 @@
+import { Flex, LoadingIndicator } from '@webkom/lego-bricks';
+import { usePreparedEffect } from '@webkom/react-prepare';
+import moment from 'moment';
+import { Link, useParams } from 'react-router-dom';
+import { fetchThread } from 'app/actions/ForumActions';
+import { CommentView } from 'app/components/Comments';
+import { Content, ContentMain } from 'app/components/Content';
+import NavigationTab, { NavigationLink } from 'app/components/NavigationTab';
+import { selectCommentsByIds, selectThreadsById } from 'app/reducers/threads';
+import { useUserContext } from 'app/routes/app/AppRoute';
+import { useAppDispatch, useAppSelector } from 'app/store/hooks';
+import type Comment from 'app/store/models/Comment';
+import type { DetailedThread } from 'app/store/models/Forum';
+
+const ThreadDetail = () => {
+  const { threadId } = useParams<{ threadId: string }>();
+  const dispatch = useAppDispatch();
+  const user = useUserContext();
+  usePreparedEffect(
+    'fetchDetailForum',
+    () => threadId && dispatch(fetchThread(parseInt(threadId ?? '0'))),
+    [threadId]
+  );
+
+  const thread: DetailedThread = useAppSelector((state) =>
+    selectThreadsById(state, { threadId })
+  );
+
+  const comments: Comment[] = useAppSelector((state) =>
+    selectCommentsByIds(state, thread ? thread.comments : [])
+  );
+
+  const fetchingThreads = useAppSelector((state) => state.threads.fetching);
+  const fetchingComments = useAppSelector((state) => state.comments.fetching);
+
+  return (
+    thread && (
+      <LoadingIndicator loading={fetchingThreads && fetchingComments}>
+        <Content>
+          <ContentMain>
+            <NavigationTab
+              back={{
+                label: 'Tilbake til liste',
+                path: `/forum/${thread.forum}`,
+              }}
+            >
+              {thread.createdBy?.id === user.currentUser.id && (
+                <NavigationLink to={`/forum/threads/${threadId}/edit`}>
+                  Rediger
+                </NavigationLink>
+              )}
+            </NavigationTab>
+
+            <h1> {thread.title}</h1>
+            {thread.description}
+            <h4>
+              Tråd startet av av{' '}
+              <Link to={`/users/${thread.createdBy?.username}`}>
+                {thread.createdBy?.username}
+              </Link>{' '}
+              den {moment(thread?.createdAt).format('lll')}
+            </h4>
```

Comment:
I don't think this should be a header

Reply:
Also, it seems `createdBy` can be undefined, and in that case it will just say "Tråd startet av av  den x". Check for `createdBy` first